ui/pages/Dashboard/components/ProjectsTable.jsx

```diff
@@ -61,7 +61,7 @@ const COLUMNS = [
     content: 'Name',
     format: project => (
       <div>
-        <Link to={`/project/${project.projectGuid}/project_page`}>{project.name}</Link>
+        <Link to={`/project/${project.projectGuid}/project_page`}>{ project.workspace ? `AnVIL: ${project.name}` : project.name}</Link>
```

Comment:
this is helpful for debugging but we don't want this in production. Maybe add a column for workspace name so we can show the workspace if it exists? Or if the workspace identifier isn't human-readable, just make a column with a checkmark for whether or not there is a corresponding workspace? Or maybe a link to the workspace?

Reply:
We can discuss this issue.